test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -197,6 +197,27 @@ def test_mobilenet_v2_with_external_data(self):
         self.run_model_test_with_external_data(model, x, rtol=1e-3, atol=1e-5,
                                                ort_optim_on=False)
 
+    @skipIfUnsupportedMinOpsetVersion(9)  # Because external data format was released with Opset 9.
+    def test_attribute_with_external_data(self):
+        class LargeModel(torch.nn.Module):
+            def forward(self, x):
+                return x + torch.ones(2, 1024)
```

Comment:
Which attribute is ONNX node is this using for export?

Reply:
attribute value_t of Constant node.